### DISCUSSION TITLE
ABP Framework with WPF

### DISCUSSION THREAD

**Discussion by ShifanilaSenthil** [abpframework/abp#19072] [Q&A]
Can we create a new application with Wpf Framework, with Tiered  and Mobile Application? If so, what is the command cli for the same
Itried this command but it didn't work 
abp new myprojectname -t wpf -u mvc --tiered -d ef -m react-native
![image](https://github.com/abpframework/abp/assets/106966267/15fb97ed-75e2-4557-b113-e3f13d7ee3b8)

**Comment by maliming** [comment]
hi

We only have a WPF startup project.

https://github.com/abpframework/abp/tree/dev/templates/wpf
